buildpipeline/dotnet-standard-win.json

```diff
@@ -144,7 +144,8 @@
       "value": "real"
     },
     "SourceBranch": {
-      "value": "master"
+      "value": "master",
+      "allowOverride": true
```

Comment:
I think https://github.com/dotnet/corefx/blob/release/2.0.0/buildpipeline/DotNet-CoreFx-Trusted-Windows.json#L54 is why we do the right thing in corefx.